pkg/apiserver/filters/requestinfo.go

```diff
@@ -76,7 +76,7 @@ func WithRequestInfo(handler http.Handler, resolver request.RequestInfoResolver)
 
 func injectToken(req *http.Request) (requestWithTokenContext *http.Request) {
 	authorization := req.Header.Get("Authorization")
-	token := strings.ReplaceAll(authorization, "bearer ", "")
+	token := strings.ReplaceAll(authorization, "Bearer ", "")
```

Comment:
This is excellent. But the README file is still incorrect. See also https://github.com/kubesphere-sigs/ks-devops#apis

Reply:
Good catch. I'll rectify it soon.

Reply:
Done.